packages/cli/src/factories/rule-exclusion.ts

```diff
@@ -3,10 +3,8 @@
 export class RuleExclusion {
     public accessibilityRuleExclusionList = [
```

Comment:
is this file meant to be duplicated at service-configuration?

Reply:
This is independent from service, just to be used by CLI tool